xbmc/addons/AddonInstaller.cpp

```diff
@@ -882,5 +882,5 @@ void CAddonUnInstallJob::ClearFavourites()
   }
 
   if (bSave)
-    CFavouritesDirectory::Save(items);
+    CServiceBroker::GetFavouritesService().GetAll(items);
```

Comment:
Shouldn't this read <code>CServiceBroker::GetFavouritesService().Save(items);</code>?